app/models/dashboard.rb

```diff
@@ -4,18 +4,34 @@ def initialize(user, defence_requests_scoped_by_policy)
     @defence_requests_scoped_by_policy = defence_requests_scoped_by_policy
   end
 
-  def defence_requests
-    ordered_defence_requests
-  end
-
   def user_role
     user.roles.uniq.first
   end
 
+  def defence_requests(type)
+    type == :active ? active_defence_requests : closed_defence_requests
+  end
+
   private
 
   attr_reader :user, :defence_requests_scoped_by_policy
 
+  def active_defence_requests
+    if user_role == "solicitor"
+      scoped_defence_requests([:not_finished])
+    else
+      ordered_defence_requests
+    end
+  end
```

Comment:
This should be done in the Policy, not here. The policy has responsibility for enforcing constraints on which defence requests a user can view